framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -251,32 +251,30 @@ private void executeLimitZeroQuery(String query) throws IOException, SQLExceptio
 		return;
 	}
 	
-	BufferedWriter writer = new BufferedWriter(new FileWriter(new File(outputFilename),true));
-	
+	BufferedWriter writer = new BufferedWriter(new FileWriter(new File(outputFilename),true));	
     Statement statement = connection.createStatement();
     ResultSet resultSet = statement.executeQuery(query);
-    
     List columnLabels = new ArrayList<String>();
-    List<Integer> types = Lists.newArrayList();
-    List<Integer> nullability = Lists.newArrayList();
+    List<Integer> columnTypes = Lists.newArrayList();
+    List<Integer> columnNullabilities = Lists.newArrayList();
+    
     try {
       int columnCount = resultSet.getMetaData().getColumnCount();
       for (int i = 1; i <= columnCount; i++) {
         columnLabels.add(resultSet.getMetaData().getColumnLabel(i));
-        types.add(resultSet.getMetaData().getColumnType(i));
-        nullability.add(resultSet.getMetaData().isNullable(i));
+        columnTypes.add(resultSet.getMetaData().getColumnType(i));
+        columnNullabilities.add(resultSet.getMetaData().isNullable(i));
       }
       
-      String msg = "\n" + query + "\n" + columnLabels + "\n"
-    		  + this.columnLabels + "\n"
-    		  + "\n" + Utils.getTypesInStrings(types) + "\n"
-    		  + Utils.getTypesInStrings(this.types) + "\n"
-    		  + "\n" + nullability + "\n"
-    		  + this.nullability + "\n";
+      String msg = "\n" + query + "\n" + columnLabels + "\n" + this.columnLabels + "\n"
```

Comment:
Suggestion: can we add some descriptors to the message so we know which query columnLabels and columnNullabilities correspond to?